src/style/App.css

```diff
@@ -28,16 +30,74 @@ body {
   border-bottom: solid 1px var(--gray);
 }
 
+.nav-brand {
+  color: var(--skyblue);
+  font-weight: 700;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+nav {
+  flex: 1;
+  padding: 0.5rem 1rem;
+}
```

Comment:
Kindly adjust the background of the website to match the design guide. A look at the guide shows the book cards and the navigation are in a container having some shadow and in turn sitting on the page with some margin. Kindly implement this too.

Your design:
![image](https://user-images.githubusercontent.com/20554583/216527536-e4ea0825-4cf3-48da-a57f-24ed4c8cc54c.png)

Design guide:
![image](https://user-images.githubusercontent.com/20554583/216527567-63e86631-2b41-4a2d-a023-4ffc427fe05e.png)